openapi.go

```diff
@@ -425,12 +425,19 @@ func parseStructTags(t reflect.Type, schemaRef *openapi3.SchemaRef) {
 		example, ok := field.Tag.Lookup("example")
 		if ok {
 			propertyValue.Example = example
-			if propertyValue.Type.Is(openapi3.TypeInteger) {
+			switch {
+			case propertyValue.Type.Is(openapi3.TypeInteger):
 				exNum, err := strconv.Atoi(example)
```

Comment:
Also, is there anything in the norm about allowing non base 10 integer?

Like an hexa, binary, octal... representation of a number ?

If so, you can also report an error maybe, or handling them with calling ParseInt(s, 0, 64) instead of Atoi

Reply:
Yeah I'll need to look at the OpenApi spec to see how this is handled